### PR TITLE
simplify getting-started; update news

### PR DESCRIPTION
**Pull request by sonichi**
## Why are these changes needed?

1. Make it easier to get started.
2. bump version to 0.2.21
3. update news

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,23 +12,25 @@
     <img src="https://github.com/microsoft/autogen/blob/main/website/static/img/flaml.svg"  width=200>
     <br>
 </p> -->
+:fire: Mar 26: Andrew Ng gave a shoutout to AutoGen in [What's next for AI agentic workflows](https://youtu.be/sal78ACtGTc?si=JduUzN_1kDnMq0vF) at Sequoia Capital's AI Ascent.
```
Comment (WaelKarkoub) [inline README.md:15]:
```suggestion
:fire: Mar 26: Andrew Ng gave a shoutout to AutoGen in ["What's next for AI agentic workflows"](https://youtu.be/sal78ACtGTc?si=JduUzN_1kDnMq0vF) at Sequoia Capital's AI Ascent.
```
Might need quotes, not certain

File: website/docs/Getting-Started.mdx [hunk 2]
```diff
@@ -103,7 +85,24 @@ with DockerCommandLineCodeExecutor(work_dir=work_dir) as code_executor:
...
+from autogen import AssistantAgent, UserProxyAgent
+
+llm_config = {"model": "gpt-4", "api_key": os.environ["OPENAI_API_KEY"]}
+assistant = AssistantAgent("assistant", llm_config=llm_config)
+user_proxy = UserProxyAgent("user_proxy", code_execution_config=False)
+
+# Start the chat
+user_proxy.initiate_chat(
+    assistant,
+    message="Tell me a joke about NVDA and TESLA stock price.",
```
Comment (WaelKarkoub) [inline website/docs/Getting-Started.mdx:101]:
```suggestion
    message="Tell me a joke about NVDA and TESLA stock prices.",
```

File: website/docs/Getting-Started.mdx [hunk 3]
```diff
@@ -103,7 +85,24 @@ with DockerCommandLineCodeExecutor(work_dir=work_dir) as code_executor:
 
 Open `coding/plot.png` to see the generated plot.
 
-  </TabItem>
+    </TabItem>
+    <TabItem value="local" label="No code execution" default>
```
Comment (jackgerrits) [inline website/docs/Getting-Started.mdx:89]: Value needs to change

```suggestion
    <TabItem value="nocode" label="No code execution" default>
```

File: autogen/coding/docker_commandline_code_executor.py [hunk 4]
```diff
@@ -84,7 +84,7 @@ def __init__(
             work_dir = Path(work_dir)
 
         if not work_dir.exists():
-            raise ValueError(f"Working directory {work_dir} does not exist.")
+            work_dir.mkdir(exist_ok=True)
```
Comment (WaelKarkoub) [inline autogen/coding/docker_commandline_code_executor.py:87]: Is `exist_ok=True` necessary if we already know the `work_dir` doesn't exist?

File: autogen/coding/local_commandline_code_executor.py [hunk 5]
```diff
@@ -57,7 +54,7 @@ def __init__(
             work_dir = Path(work_dir)
 
         if not work_dir.exists():
-            raise ValueError(f"Working directory {work_dir} does not exist.")
+            work_dir.mkdir(exist_ok=True)
```
Comment (WaelKarkoub) [inline autogen/coding/local_commandline_code_executor.py:57]: Same thing here

**Review by WaelKarkoub — APPROVED** [review]
LGTM

**Pull request merged**